docs/articles_en/learn-openvino/llm_inference_guide/llm-inference-hf.rst

```diff
@@ -276,9 +276,10 @@ includes **Dynamic quantization** of activations of 4/8-bit quantized MatMuls an
          ov_config={"KV_CACHE_PRECISION": "u8", "DYNAMIC_QUANTIZATION_GROUP_SIZE": "32", "PERFORMANCE_HINT": "LATENCY"}
      )
 
-.. note::
+  .. note::
+     Currently, for KV-cache quantization, GPU ignores the DYNAMIC_QUANTIZATION_GROUP_SIZE property, using ``group_size = head_size``. Additionally, it does not support the ``get_state()`` and ``set_state()`` APIs when KV-cache quantization is enabled.
 
-   Currently, both Dynamic quantization and KV-cache quantization are available for CPU device.
+     For GPU, KV-cache quantization is enabled by default on all non-systolic platforms, and can be disabled by setting KV_CACHE_PRECISION to ``undefined``.
```

Comment:
I think this `non-systolic` term may be not clear for external people, so I'd suggest replacing it with smth like `platforms without XMX support`.